features/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(SUBSYS_NAME features)
 set(SUBSYS_DESC "Point cloud features library")
-set(SUBSYS_DEPS common search kdtree octree filters 2d)
+set(SUBSYS_DEPS common sample_consensus search kdtree octree filters 2d)
```

Comment:
@SergioRAgostinho I hope this new dependency won't break your CI job distribution :)